server/elastic/src/__tests__/adapter.test.ts

```diff
@@ -38,7 +37,22 @@ describe('client', () => {
     console.log(hits)
   })
 
-  // it('should find document', async () => {
-  //   const adapter = await createElasticAdapter('http://localhost:9200/', 'ws1')
-  // })
+  it('should find document with raw search', async () => {
```

Comment:
We do not start elastic on tests, so we need to add it before we could write tests for elastic.

Reply:
Oh I saw we had one test and just thought it would be nice to add more. Are there any problems with elastic in test runner? Or should I just add it there?

Reply:
You need to continue work on search API, since right now it is pure elastic, but it should be our minimal APIs suitable for our search. And in case we could like to replace elastic in future we will be able to do so with just adapter replacement.

Reply:
No problem, we just not added it.